pkg/virt-controller/watch/migration.go

```diff
@@ -576,6 +594,19 @@ func (c *MigrationController) handleTargetPodHandoff(migration *virtv1.VirtualMa
 	// the vmi and prepare the local environment for the migration
 	vmiCopy.ObjectMeta.Labels[virtv1.MigrationTargetNodeNameLabel] = pod.Spec.NodeName
 
+	if controller.VMIHasHotplugVolumes(vmiCopy) {
+		attachmentPods, err := controller.AttachmentPods(pod, c.podInformer)
+		if err != nil {
+			return fmt.Errorf("failed to get attachment pods: %v", err)
+		}
+		if len(attachmentPods) > 0 {
+			log.Log.Object(migration).Infof("Target attachment pod for vmi %s/%s: %s", vmiCopy.Namespace, vmiCopy.Name, string(attachmentPods[0].UID))
+			vmiCopy.ObjectMeta.Labels[virtv1.MigrationTargetAttachmentPodLabel] = string(attachmentPods[0].UID)
```

Comment:
Is there a reason this needs to be a label? I'd think this would be something we'd store on the vmi.Status.MigrationState struct. 

we use a label for the NodeName because we have an informer that is using that label as a selector

Reply:
No specific reason. Right, probably the struct would be a better place.